CDTDatastore/touchdb/TDReplicator.m

```diff
@@ -765,13 +768,52 @@ - (void)fetchRemoteCheckpointDoc
                     if (error.code == kTDStatusNotFound) [self maybeCreateRemoteDB];
                     response = $castIf(NSDictionary, response);
                     self.remoteCheckpoint = response;
-                    NSObject* remoteLastSequence = response[@"lastSequence"];
+                    NSObject* remoteLastSequence = response[@"source_last_seq"];
+                    if (!remoteLastSequence) {
+                        // the checkpoint doc is the old format, it will be converted when we
+                        // resave, for now
+                        // fall back to the `lastSequence` property
+                        remoteLastSequence = response[@"seq"];
```

Comment:
https://github.com/cloudant/CDTDatastore/pull/331/files#diff-7cda42788bca18e1e304c7a20a5c7c72R85 seems to suggest it may be called `seq`